tests/phpunit/tests/style-engine/styleEngine.php

```diff
@@ -654,7 +654,7 @@ public function test_should_return_stylesheet_from_css_rules() {
 	/**
 	 * Tests that incoming styles are deduped and merged.
 	 *
-	 * @ticket 56467
+	 * @ticket 58811, 56467
```

Comment:
```suggestion
	 * @ticket 58811
	 * @ticket 56467
```

Correct `@ticket` annotation formate

Reply:
Done, thanks for the suggestion :)